Documents/OrderingRules.md

```diff
@@ -0,0 +1,131 @@
+# Ordering Rules
+
+## The Abstract
+
+Rules which enforce a standard ordering scheme for code contents.
+
+## Rules
+
+Within a class, struct or interface
+
+* Constant Fields
+* Fields
+* Constructors
+* Finalizers (Destructors)
+* Delegates
+* Enums
```

Comment:
Yes. This is only for a sorting in the same file. Nested classes or enums for me are ugly.